src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping/edit-keyboard-mapping.component.ts

```diff
@@ -97,4 +103,38 @@ export class EditKeyboardMappingComponent implements OnInit {
   public cancel(): void {
     this.onCancel.emit()
   }
+
+  public get saveButtonErrorsTooltip(): string | undefined {
```

Comment:
Instead of naming the getter method based on where it is used, it should state what it does/contains. A suggestion could be `invalidControlsErrorMessage`.

The undefined type can be removed as well. An empty string should be fine use when there are no errors.

The same goes for the other form component(s).